ehrql/tables/beta/tpp.py

```diff
@@ -85,51 +85,67 @@ def for_patient_on(self, date):
 
 @table
 class apcs(EventFrame):
+    """
+    Admitted patient care spells (APCS) data is provided via the NHS Secondary Uses Service.
+
+    This table gives core details of spells.
+
+    Each row is an in-hospital spell: a period of continuous care within a single trust.
```

Comment:
comment: This is from the current [OpenSAFELY docs](https://docs.opensafely.org/data-sources/apc/).